.changeset/little-keys-stare.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-catalog-backend-module-aws': patch
```

Comment:
I'd be ok to skip the changeset since it'll be covered by the previous one

Reply:
removed